components/cam/bld/build-namelist

```diff
@@ -3979,7 +3979,7 @@ if ($cfg->get('scam')) {
 }
 
 # CAM generates IOP file for SCAM
-if ($cfg->get('camiop')) {
+if ($cfg->get('e3smreplay')) {
     add_default($nl, 'inithist', 'val'=>'CAMIOP');
```

Comment:
About the 'CAMIOP' type of inithist: do we want to rename this type to reflect the current naming convention for this feature? If changed, it would trigger changes in several other files (including configure, build-namelist, namelist_definition.xml, and cam_history.F90). It can certainly be considered in a future PR.